Header header
string[] joint_names
string json_representation
duration rate
